packages/core-web/package.json

```diff
@@ -13,9 +13,7 @@
   "main": "./dist/index.cjs",
   "module": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "files": [
-    "dist"
-  ],
+  "files": ["dist"],
```

Comment:
@vcashwin any idea why my environment has made this format change, shall we just accept it?

Reply:
Ah seems like our versioning job then changes the formatting back e.g. https://github.com/livepeer/ui-kit/commit/e01f14c454995de1fa878d3b02dd73c94dd8944e